src/main/java/edu/greenblitz/robotName/utils/shootingCalculations/ShootingZone.java

```diff
@@ -0,0 +1,45 @@
+package edu.greenblitz.robotName.utils.shootingCalculations;
+
+import edu.greenblitz.robotName.shootingStateService.ShootingState;
+import edu.greenblitz.robotName.utils.GBMath;
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+
+import java.util.List;
+
+public class ShootingZone extends GBMath.GBCircle {
+    private List<Pair<Translation2d, Translation2d>> restrictedBounds; //first min, second max
+
+    public ShootingZone(Translation2d translation, double radius) {
+        super(translation, radius);
+    }
```

Comment:
im pretty sure you dont need constructor that calls the super constructor.

you can just delete this